updateHosts.sh

```diff
@@ -50,6 +40,37 @@ updateHosts() {
 	flushDNS
 }
 
+defaultHosts()
+{
+cat <<EOF
+
+# IPv4
+
+127.0.0.1 localhost
+127.0.0.1 localhost.localdomain
+127.0.0.1 local
+255.255.255.255 broadcasthost
+
+# IPv6
+
+::1 localhost
+fe80::1%lo0 localhost
+
+EOF
+}
+
+# Restore default Hosts file.
+restore()
+{
+  OUT=$(defaultHosts)
+  echo "* * * Restoring default hosts * * *"
+  echo "This is restored default hosts file: $OUT" > hosts
+	sudo cp hosts /etc/hosts
+  flushDNS
+}
+
```

Comment:
the objective is revert back host file to OS default , one that come of freshly loaded OS.
Similar Ad-away does for android devices , time to time have found it  extremely useful.

i have used template from Microsoft website and cross checked on Mac OSX too.